setup.py

```diff
@@ -17,7 +17,7 @@ def clean_html(raw_html):
     return cleantext
 
 
-def _get_version():
+def get_version():
     # get version string from version.py
     version_file = os.path.join(os.path.dirname(__file__), "version.py")
```

Comment:
do we leave the version same as old one?

Reply:
You mean the version number in [`version.py`](https://github.com/facebookresearch/multimodal/blob/main/version.py#L17)? We can change it to beta release but until we decide to do a stable release the version number won't show up in PyPI.